js/graphql.md

```diff
@@ -0,0 +1,1346 @@
+---
+---
+# The GraphQL Transform
+
+The GraphQL Transform is a library that simplifies the process of developing, deploying, and maintaining GraphQL APIs. With it, you define your API using the GraphQL Schema Definition Language (SDL) and can then use this library where to transform it into a fully descriptive cloudformation template that implements the API's data model.
```

Comment:
"After defining your API using the GraphQL Schema Definition Language (SDL), you can then use this library to transform it into a fully descriptive CloudFormation template that implements the API's data model."

Grammatical opinions and capitalization (we should be consistent with official spelling/typesetting of things).